src/reducers/paymentMethods.js

```diff
@@ -82,11 +88,39 @@ const wyre = (state, action) => {
           ...action.payload.config
         },
       };
+    case 'GET_ACTIVE_TRANSACTION':
```

Comment:
Why is this called GET_ACTIVE_TRANSACTION, we need to mention the RATES, because it's fetching rates not an active transaction